server/data/vocabularies.json

```diff
@@ -0,0 +1,2246 @@
+            }
+        ]
+    },
+    {
+        "_id": "asset_formats",
+        "display_name": "Asset formats",
+        "type": "manageable",
+        "unique_field": "qcode",
+        "items": [
+            {
+                "name": "Avi",
+                "qcode": "Avi",
+                "is_active": true
+            },
+            {
+                "name": "Gif",
+                "qcode": "Gif",
+                "is_active": true
+            },
+            {
+                "name": "Jpeg",
+                "qcode": "Jpeg",
+                "is_active": true
+            },
+            {
+                "name": "MSExcel",
+                "qcode": "MSExcel",
+                "is_active": true
+            },
+            {
+                "name": "MSExcelx",
+                "qcode": "MSExcelx",
+                "is_active": true
+            },
+            {
+                "name": "MSPowerpoint",
+                "qcode": "MSPowerpoint",
+                "is_active": true
+            },
+            {
+                "name": "MSPowerpointx",
+                "qcode": "MSPowerpointx",
+                "is_active": true
+            },
+            {
+                "name": "MSWord",
+                "qcode": "MSWord",
+                "is_active": true
+            },
+            {
+                "name": "MSWordx",
+                "qcode": "MSWordx",
+                "is_active": true
+            },
+            {
+                "name": "Mov",
+                "qcode": "Mov",
+                "is_active": true
+            },
+            {
+                "name": "Mpeg2",
+                "qcode": "Mpeg2",
+                "is_active": true
+            },
+            {
+                "name": "Mpeg3",
+                "qcode": "Mpeg3",
+                "is_active": true
+            },
+            {
+                "name": "Mpeg4",
+                "qcode": "Mpeg4",
+                "is_active": true
+            },
+            {
+                "name": "Pdf",
+                "qcode": "Pdf",
+                "is_active": true
+            },
+            {
+                "name": "Ping",
+                "qcode": "Ping",
+                "is_active": true
+            },
+            {
+                "name": "Table",
+                "qcode": "Table",
+                "is_active": true
+            },
+            {
+                "name": "Text",
+                "qcode": "Text",
+                "is_active": true
+            },
+            {
+                "name": "Wma",
+                "qcode": "Wma",
+                "is_active": true
+            },
+            {
+                "name": "Wmv",
+                "qcode": "Wmv",
+                "is_active": true
+            },
+            {
+                "name": "Zip",
+                "qcode": "Zip",
+                "is_active": true
+            }
+        ]
+    },
+    {
+        "_id": "asset_types",
```

Comment:
I'm not sure whether we need `asset_types` cv at all.

Reply:
indeed, checked with @karelpetrak and will remove them, thanks